README.md

```diff
@@ -37,9 +43,8 @@ Select 1:
 
 Describe the rules of scope in JavaScript.
 
-Your Answer:
-```text
-```
+Your Answer: Scope refers to where variables and functions are accessible, and in what context it is being exicuted. Scopes can be global or local. Variables have functions scopen and functions have the same scope as variables.
+
 
```

Comment:
On the right track with that first sentence. What do you mean by the last one? Important to remember that each function creates its own scope that cannot be accessed by anything outside of it, unless you pass something through that function's return value.